[INFO] [2021-06-13 17:43:01,108] [7:139672034457344] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/host/9998/create
[INFO] [2021-06-13 17:43:01,327] [7:139672034457344] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:01,327] [7:139672034457344] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/host/9998/create {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:01,327] [7:139672034457344] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/guest/9999/create
[INFO] [2021-06-13 17:43:01,959] [7:139672034457344] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:01,960] [7:139672034457344] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/guest/9999/create {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:03,212] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/host/9998/resource/apply
[INFO] [2021-06-13 17:43:03,240] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:03,241] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/host/9998/resource/apply {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:03,241] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/guest/9999/resource/apply
[INFO] [2021-06-13 17:43:03,263] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:03,263] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/guest/9999/resource/apply {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:03,268] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/host/9998/start
[INFO] [2021-06-13 17:43:03,309] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:03,310] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/host/9998/start {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:03,310] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/guest/9999/start
[INFO] [2021-06-13 17:43:03,350] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:03,350] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/guest/9999/start {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:03,511] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/status/running
[INFO] [2021-06-13 17:43:03,546] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:03,546] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/status/running {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:03,547] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/status/running
[INFO] [2021-06-13 17:43:03,571] [7:139672144258816] - api_utils.py[line:129]: {"retcode":103,"retmsg":"update task status failed"}

[INFO] [2021-06-13 17:43:03,571] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/status/running {'retcode': 103, 'retmsg': 'update task status failed'}
[INFO] [2021-06-13 17:43:03,593] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/start
[INFO] [2021-06-13 17:43:03,673] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:03,673] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/start {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:03,674] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/start
[INFO] [2021-06-13 17:43:03,750] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:03,750] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/start {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:05,781] [145:140298433996608] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/report
[INFO] [2021-06-13 17:43:05,856] [145:140298433996608] - api_utils.py[line:129]: {"retcode":103,"retmsg":"update task status failed"}

[INFO] [2021-06-13 17:43:05,857] [145:140298433996608] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/report {'retcode': 103, 'retmsg': 'update task status failed'}
[INFO] [2021-06-13 17:43:05,913] [147:140166847178560] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/report
[INFO] [2021-06-13 17:43:05,955] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/collect
[INFO] [2021-06-13 17:43:05,981] [7:139672144258816] - api_utils.py[line:129]: {"data":{"elapsed":null,"end_time":null,"job_id":"202106131743010453261","party_id":"9998","party_status":"running","role":"host","start_time":1623606183616,"status":"running","task_id":"202106131743010453261_reader_1","task_version":0,"update_time":1623606181197},"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:05,982] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/collect {'data': {'elapsed': None, 'end_time': None, 'job_id': '202106131743010453261', 'party_id': '9998', 'party_status': 'running', 'role': 'host', 'start_time': 1623606183616, 'status': 'running', 'task_id': '202106131743010453261_reader_1', 'task_version': 0, 'update_time': 1623606181197}, 'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:05,983] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/collect
[INFO] [2021-06-13 17:43:06,005] [7:139672144258816] - api_utils.py[line:129]: {"data":{"elapsed":null,"end_time":null,"job_id":"202106131743010453261","party_id":"9999","party_status":"running","role":"guest","start_time":1623606183700,"status":"running","task_id":"202106131743010453261_reader_1","task_version":0,"update_time":1623606181392},"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:06,006] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/collect {'data': {'elapsed': None, 'end_time': None, 'job_id': '202106131743010453261', 'party_id': '9999', 'party_status': 'running', 'role': 'guest', 'start_time': 1623606183700, 'status': 'running', 'task_id': '202106131743010453261_reader_1', 'task_version': 0, 'update_time': 1623606181392}, 'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:06,012] [147:140166847178560] - api_utils.py[line:129]: {"retcode":103,"retmsg":"update task status failed"}

[INFO] [2021-06-13 17:43:06,012] [147:140166847178560] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/report {'retcode': 103, 'retmsg': 'update task status failed'}
[INFO] [2021-06-13 17:43:06,379] [145:140298433996608] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/report
[INFO] [2021-06-13 17:43:06,422] [145:140298433996608] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:06,423] [145:140298433996608] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/report {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:06,424] [145:140298433996608] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/report
[INFO] [2021-06-13 17:43:06,458] [145:140298433996608] - api_utils.py[line:129]: {"retcode":103,"retmsg":"update task status failed"}

[INFO] [2021-06-13 17:43:06,458] [145:140298433996608] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/report {'retcode': 103, 'retmsg': 'update task status failed'}
[INFO] [2021-06-13 17:43:06,500] [147:140166847178560] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/report
[INFO] [2021-06-13 17:43:06,550] [147:140166847178560] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:06,550] [147:140166847178560] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/report {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:06,552] [147:140166847178560] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/report
[INFO] [2021-06-13 17:43:06,590] [147:140166847178560] - api_utils.py[line:129]: {"retcode":103,"retmsg":"update task status failed"}

[INFO] [2021-06-13 17:43:06,591] [147:140166847178560] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/report {'retcode': 103, 'retmsg': 'update task status failed'}
[INFO] [2021-06-13 17:43:08,225] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/collect
[INFO] [2021-06-13 17:43:08,241] [7:139672144258816] - api_utils.py[line:129]: {"data":{"elapsed":792,"end_time":1623606186379,"job_id":"202106131743010453261","party_id":"9998","party_status":"failed","role":"host","start_time":1623606183616,"status":"running","task_id":"202106131743010453261_reader_1","task_version":0,"update_time":1623606181197},"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:08,242] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/collect {'data': {'elapsed': 792, 'end_time': 1623606186379, 'job_id': '202106131743010453261', 'party_id': '9998', 'party_status': 'failed', 'role': 'host', 'start_time': 1623606183616, 'status': 'running', 'task_id': '202106131743010453261_reader_1', 'task_version': 0, 'update_time': 1623606181197}, 'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:08,242] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/collect
[INFO] [2021-06-13 17:43:08,257] [7:139672144258816] - api_utils.py[line:129]: {"data":{"elapsed":844,"end_time":1623606186500,"job_id":"202106131743010453261","party_id":"9999","party_status":"failed","role":"guest","start_time":1623606183700,"status":"running","task_id":"202106131743010453261_reader_1","task_version":0,"update_time":1623606181392},"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:08,258] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/collect {'data': {'elapsed': 844, 'end_time': 1623606186500, 'job_id': '202106131743010453261', 'party_id': '9999', 'party_status': 'failed', 'role': 'guest', 'start_time': 1623606183700, 'status': 'running', 'task_id': '202106131743010453261_reader_1', 'task_version': 0, 'update_time': 1623606181392}, 'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:08,348] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/status/failed
[INFO] [2021-06-13 17:43:08,756] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:08,756] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/status/failed {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:08,757] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/status/failed
[INFO] [2021-06-13 17:43:08,798] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:08,799] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/status/failed {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:08,816] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/stop/failed
[INFO] [2021-06-13 17:43:08,859] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:08,859] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/stop/failed {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:08,859] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/stop/failed
[INFO] [2021-06-13 17:43:08,914] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:08,914] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/stop/failed {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:09,039] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/host/9998/status/running
[INFO] [2021-06-13 17:43:09,068] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:09,069] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/host/9998/status/running {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:09,069] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/guest/9999/status/running
[INFO] [2021-06-13 17:43:09,090] [7:139672144258816] - api_utils.py[line:129]: {"retcode":103,"retmsg":"update task status failed"}

[INFO] [2021-06-13 17:43:09,091] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/guest/9999/status/running {'retcode': 103, 'retmsg': 'update task status failed'}
[INFO] [2021-06-13 17:43:09,113] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/host/9998/start
[INFO] [2021-06-13 17:43:09,253] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:09,253] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/host/9998/start {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:09,254] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/guest/9999/start
[INFO] [2021-06-13 17:43:09,888] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:09,889] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/guest/9999/start {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:09,896] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/host/9998/update
[INFO] [2021-06-13 17:43:09,949] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:09,950] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/host/9998/update {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:09,950] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/guest/9999/update
[INFO] [2021-06-13 17:43:09,980] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:09,980] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/guest/9999/update {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:11,549] [192:139662190929728] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/host/9998/report
[INFO] [2021-06-13 17:43:11,588] [192:139662190929728] - api_utils.py[line:129]: {"retcode":103,"retmsg":"update task status failed"}

[INFO] [2021-06-13 17:43:11,588] [192:139662190929728] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/host/9998/report {'retcode': 103, 'retmsg': 'update task status failed'}
[INFO] [2021-06-13 17:43:11,652] [201:140572012922688] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/guest/9999/report
[INFO] [2021-06-13 17:43:11,692] [201:140572012922688] - api_utils.py[line:129]: {"retcode":103,"retmsg":"update task status failed"}

[INFO] [2021-06-13 17:43:11,692] [201:140572012922688] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/guest/9999/report {'retcode': 103, 'retmsg': 'update task status failed'}
[INFO] [2021-06-13 17:43:12,068] [192:139662190929728] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/host/9998/report
[INFO] [2021-06-13 17:43:12,588] [192:139662190929728] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:12,589] [192:139662190929728] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/host/9998/report {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:12,592] [192:139662190929728] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/host/9998/report
[INFO] [2021-06-13 17:43:12,631] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/host/9998/collect
[INFO] [2021-06-13 17:43:12,651] [7:139672144258816] - api_utils.py[line:129]: {"data":{"elapsed":736,"end_time":1623606192067,"job_id":"202106131743010453261","party_id":"9998","party_status":"failed","role":"host","start_time":1623606189172,"status":"running","task_id":"202106131743010453261_reader_0","task_version":0,"update_time":1623606181208},"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:12,652] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/host/9998/collect {'data': {'elapsed': 736, 'end_time': 1623606192067, 'job_id': '202106131743010453261', 'party_id': '9998', 'party_status': 'failed', 'role': 'host', 'start_time': 1623606189172, 'status': 'running', 'task_id': '202106131743010453261_reader_0', 'task_version': 0, 'update_time': 1623606181208}, 'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:12,652] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/guest/9999/collect
[INFO] [2021-06-13 17:43:12,668] [201:140572012922688] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/guest/9999/report
[INFO] [2021-06-13 17:43:12,678] [7:139672144258816] - api_utils.py[line:129]: {"data":{"elapsed":null,"end_time":null,"job_id":"202106131743010453261","party_id":"9999","party_status":"running","role":"guest","start_time":1623606189277,"status":"running","task_id":"202106131743010453261_reader_0","task_version":0,"update_time":1623606181823},"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:12,680] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/guest/9999/collect {'data': {'elapsed': None, 'end_time': None, 'job_id': '202106131743010453261', 'party_id': '9999', 'party_status': 'running', 'role': 'guest', 'start_time': 1623606189277, 'status': 'running', 'task_id': '202106131743010453261_reader_0', 'task_version': 0, 'update_time': 1623606181823}, 'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:12,724] [192:139662190929728] - api_utils.py[line:129]: {"retcode":103,"retmsg":"update task status failed"}

[INFO] [2021-06-13 17:43:12,725] [192:139662190929728] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/host/9998/report {'retcode': 103, 'retmsg': 'update task status failed'}
[INFO] [2021-06-13 17:43:12,773] [201:140572012922688] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:12,774] [201:140572012922688] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/guest/9999/report {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:12,777] [201:140572012922688] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/guest/9999/report
[INFO] [2021-06-13 17:43:13,066] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/host/9998/status/failed
[INFO] [2021-06-13 17:43:13,086] [201:140572012922688] - api_utils.py[line:129]: {"retcode":103,"retmsg":"update task status failed"}

[INFO] [2021-06-13 17:43:13,087] [201:140572012922688] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/guest/9999/report {'retcode': 103, 'retmsg': 'update task status failed'}
[INFO] [2021-06-13 17:43:13,154] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:13,155] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/host/9998/status/failed {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:13,155] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/guest/9999/status/failed
[INFO] [2021-06-13 17:43:13,207] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:13,207] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/guest/9999/status/failed {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:13,228] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/host/9998/stop/failed
[INFO] [2021-06-13 17:43:13,291] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:13,292] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/host/9998/stop/failed {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:13,292] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/guest/9999/stop/failed
[INFO] [2021-06-13 17:43:13,346] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:13,346] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_0/202106131743010453261_reader_0/0/guest/9999/stop/failed {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:13,414] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/host/9998/update
[INFO] [2021-06-13 17:43:13,441] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:13,441] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/host/9998/update {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:13,442] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/guest/9999/update
[INFO] [2021-06-13 17:43:13,465] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:13,466] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/guest/9999/update {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:13,528] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/host/9998/model
[INFO] [2021-06-13 17:43:13,797] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:13,798] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/host/9998/model {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:13,798] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/guest/9999/model
[INFO] [2021-06-13 17:43:14,020] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:14,020] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/guest/9999/model {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:14,021] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/host/9998/status/failed
[INFO] [2021-06-13 17:43:14,081] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:14,082] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/host/9998/status/failed {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:14,083] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/guest/9999/status/failed
[INFO] [2021-06-13 17:43:14,153] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:14,153] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/guest/9999/status/failed {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:14,196] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/host/9998/stop/failed
[INFO] [2021-06-13 17:43:14,428] [7:139672144258816] - api_utils.py[line:129]: {"data":{"202106131743010453261":{"202106131743010453261_dataio_0":"success","202106131743010453261_dataio_1":"success","202106131743010453261_evaluation_0":"success","202106131743010453261_hetero_secure_boost_0":"success","202106131743010453261_intersection_0":"success","202106131743010453261_intersection_1":"success","202106131743010453261_reader_0":"success","202106131743010453261_reader_1":"success"}},"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:14,428] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/host/9998/stop/failed {'data': {'202106131743010453261': {'202106131743010453261_dataio_0': 'success', '202106131743010453261_dataio_1': 'success', '202106131743010453261_evaluation_0': 'success', '202106131743010453261_hetero_secure_boost_0': 'success', '202106131743010453261_intersection_0': 'success', '202106131743010453261_intersection_1': 'success', '202106131743010453261_reader_0': 'success', '202106131743010453261_reader_1': 'success'}}, 'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:14,429] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/guest/9999/stop/failed
[INFO] [2021-06-13 17:43:14,674] [7:139672144258816] - api_utils.py[line:129]: {"data":{"202106131743010453261":{"202106131743010453261_dataio_0":"success","202106131743010453261_dataio_1":"success","202106131743010453261_evaluation_0":"success","202106131743010453261_hetero_secure_boost_0":"success","202106131743010453261_intersection_0":"success","202106131743010453261_intersection_1":"success","202106131743010453261_reader_0":"success","202106131743010453261_reader_1":"success"}},"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:14,674] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/guest/9999/stop/failed {'data': {'202106131743010453261': {'202106131743010453261_dataio_0': 'success', '202106131743010453261_dataio_1': 'success', '202106131743010453261_evaluation_0': 'success', '202106131743010453261_hetero_secure_boost_0': 'success', '202106131743010453261_intersection_0': 'success', '202106131743010453261_intersection_1': 'success', '202106131743010453261_reader_0': 'success', '202106131743010453261_reader_1': 'success'}}, 'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:14,676] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/host/9998/clean
[INFO] [2021-06-13 17:43:14,709] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:14,710] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/host/9998/clean {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:14,710] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/guest/9999/clean
[INFO] [2021-06-13 17:43:14,741] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:14,742] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/guest/9999/clean {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:16,036] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/host/9998/stop/failed
[INFO] [2021-06-13 17:43:16,320] [7:139672144258816] - api_utils.py[line:129]: {"data":{"202106131743010453261":{"202106131743010453261_dataio_0":"success","202106131743010453261_dataio_1":"success","202106131743010453261_evaluation_0":"success","202106131743010453261_hetero_secure_boost_0":"success","202106131743010453261_intersection_0":"success","202106131743010453261_intersection_1":"success","202106131743010453261_reader_0":"success","202106131743010453261_reader_1":"success"}},"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:16,320] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/host/9998/stop/failed {'data': {'202106131743010453261': {'202106131743010453261_dataio_0': 'success', '202106131743010453261_dataio_1': 'success', '202106131743010453261_evaluation_0': 'success', '202106131743010453261_hetero_secure_boost_0': 'success', '202106131743010453261_intersection_0': 'success', '202106131743010453261_intersection_1': 'success', '202106131743010453261_reader_0': 'success', '202106131743010453261_reader_1': 'success'}}, 'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:16,321] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/guest/9999/stop/failed
[INFO] [2021-06-13 17:43:16,659] [7:139672144258816] - api_utils.py[line:129]: {"data":{"202106131743010453261":{"202106131743010453261_dataio_0":"success","202106131743010453261_dataio_1":"success","202106131743010453261_evaluation_0":"success","202106131743010453261_hetero_secure_boost_0":"success","202106131743010453261_intersection_0":"success","202106131743010453261_intersection_1":"success","202106131743010453261_reader_0":"success","202106131743010453261_reader_1":"success"}},"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:16,660] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/guest/9999/stop/failed {'data': {'202106131743010453261': {'202106131743010453261_dataio_0': 'success', '202106131743010453261_dataio_1': 'success', '202106131743010453261_evaluation_0': 'success', '202106131743010453261_hetero_secure_boost_0': 'success', '202106131743010453261_intersection_0': 'success', '202106131743010453261_intersection_1': 'success', '202106131743010453261_reader_0': 'success', '202106131743010453261_reader_1': 'success'}}, 'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:16,661] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/host/9998/clean
[INFO] [2021-06-13 17:43:16,678] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:16,679] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/host/9998/clean {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:43:16,679] [7:139672144258816] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/guest/9999/clean
[INFO] [2021-06-13 17:43:16,729] [7:139672144258816] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:43:16,729] [7:139672144258816] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/guest/9999/clean {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:46:58,102] [7:139673186752256] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/initiator/202106131743010453261/guest/9999/rerun
[INFO] [2021-06-13 17:46:58,177] [7:139672025802496] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/stop/canceled
[INFO] [2021-06-13 17:46:58,269] [7:139672025802496] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:46:58,270] [7:139672025802496] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/stop/canceled {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:46:58,270] [7:139672025802496] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/stop/canceled
[INFO] [2021-06-13 17:46:58,345] [7:139672025802496] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:46:58,346] [7:139672025802496] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/stop/canceled {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:46:58,387] [7:139672025802496] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/clean/metrics
[INFO] [2021-06-13 17:46:58,403] [7:139672025802496] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:46:58,404] [7:139672025802496] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/host/9998/clean/metrics {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:46:58,404] [7:139672025802496] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/clean/metrics
[INFO] [2021-06-13 17:46:58,419] [7:139672025802496] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:46:58,419] [7:139672025802496] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/0/guest/9999/clean/metrics {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:46:58,452] [7:139672025802496] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/1/host/9998/create
[INFO] [2021-06-13 17:46:58,497] [7:139672025802496] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:46:58,497] [7:139672025802496] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/1/host/9998/create {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:46:58,498] [7:139672025802496] - api_utils.py[line:122]: remote http api request: http://127.0.0.1:9380/v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/1/guest/9999/create
[INFO] [2021-06-13 17:46:58,516] [7:139672025802496] - api_utils.py[line:129]: {"retcode":0,"retmsg":"success"}

[INFO] [2021-06-13 17:46:58,516] [7:139672025802496] - api_utils.py[line:131]: remote http api response: /v1/party/202106131743010453261/reader_1/202106131743010453261_reader_1/1/guest/9999/create {'retcode': 0, 'retmsg': 'success'}
[INFO] [2021-06-13 17:46:58,526] [7:139673186752256] - api_utils.py[line:129]: {"retcode":100,"retmsg":"Traceback (most recent call last):\n  File \"/usr/local/lib/python3.6/site-packages/playhouse/apsw_ext.py\", line 112, in execute_sql\n    cursor.execute(sql, params or ())\n  File \"src/cursor.c\", line 236, in resetcursor\napsw.ConstraintError: ConstraintError: UNIQUE constraint failed: t_task.f_job_id, t_task.f_task_id, t_task.f_task_version, t_task.f_role, t_task.f_party_id\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/fate/python/fate_flow/operation/job_saver.py\", line 99, in create_job_family_entity\n    rows = obj.save(force_insert=True)\n  File \"/fate/python/fate_arch/storage/metastore/base_model.py\", line 89, in save\n    return super(BaseModel, self).save(*args, **kwargs)\n  File \"/usr/local/lib/python3.6/site-packages/peewee.py\", line 5964, in save\n    self.insert(**field_dict).execute()\n  File \"/usr/local/lib/python3.6/site-packages/peewee.py\", line 1737, in inner\n    return method(self, database, *args, **kwargs)\n  File \"/usr/local/lib/python3.6/site-packages/peewee.py\", line 1808, in execute\n    return self._execute(database)\n  File \"/usr/local/lib/python3.6/site-packages/peewee.py\", line 2512, in _execute\n    return super(Insert, self)._execute(database)\n  File \"/usr/local/lib/python3.6/site-packages/peewee.py\", line 2275, in _execute\n    cursor = database.execute(self)\n  File \"/usr/local/lib/python3.6/site-packages/peewee.py\", line 2888, in execute\n    return self.execute_sql(sql, params, commit=commit)\n  File \"/usr/local/lib/python3.6/site-packages/playhouse/apsw_ext.py\", line 112, in execute_sql\n    cursor.execute(sql, params or ())\n  File \"/usr/local/lib/python3.6/site-packages/peewee.py\", line 2666, in __exit__\n    reraise(new_type, new_type(*exc_args), traceback)\n  File \"/usr/local/lib/python3.6/site-packages/peewee.py\", line 179, in reraise\n    raise value.with_traceback(tb)\n  File \"/usr/local/lib/python3.6/site-packages/playhouse/apsw_ext.py\", line 112, in execute_sql\n    cursor.execute(sql, params or ())\n  File \"src/cursor.c\", line 236, in resetcursor\npeewee.IntegrityError: ConstraintError: UNIQUE constraint failed: t_task.f_job_id, t_task.f_task_id, t_task.f_task_version, t_task.f_role, t_task.f_party_id\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/lib/python3.6/site-packages/flask/app.py\", line 2292, in wsgi_app\n    response = self.full_dispatch_request()\n  File \"/usr/local/lib/python3.6/site-packages/flask/app.py\", line 1815, in full_dispatch_request\n    rv = self.handle_user_exception(e)\n  File \"/usr/local/lib/python3.6/site-packages/flask/app.py\", line 1718, in handle_user_exception\n    reraise(exc_type, exc_value, tb)\n  File \"/usr/local/lib/python3.6/site-packages/flask/_compat.py\", line 35, in reraise\n    raise value\n  File \"/usr/local/lib/python3.6/site-packages/flask/app.py\", line 1813, in full_dispatch_request\n    rv = self.dispatch_request()\n  File \"/usr/local/lib/python3.6/site-packages/flask/app.py\", line 1799, in dispatch_request\n    return self.view_functions[rule.endpoint](**req.view_args)\n  File \"/fate/python/fate_flow/scheduling_apps/initiator_app.py\", line 48, in rerun_job\n    component_name=request.json.get(\"component_name\"))\n  File \"/fate/python/fate_flow/scheduler/dag_scheduler.py\", line 363, in rerun_job\n    JobController.initialize_tasks(job_id, _role, _party_id, False, job.f_initiator_role, job.f_initiator_party_id, RunParameters(**job.f_runtime_conf_on_party[\"job_parameters\"]), dsl_parser, component_name=task.f_component_name, task_version=task.f_task_version)\n  File \"/fate/python/fate_flow/controller/job_controller.py\", line 192, in initialize_tasks\n    role=role, party_id=party_id, run_on_this_party=run_on_this_party, task_info=task_info)\n  File \"/fate/python/fate_flow/controller/task_controller.py\", line 52, in create_task\n    JobSaver.create_task(task_info=task_info)\n  File \"/fate/python/fate_flow/operation/job_saver.py\", line 36, in create_task\n    return cls.create_job_family_entity(Task, task_info)\n  File \"/usr/local/lib/python3.6/site-packages/peewee.py\", line 394, in inner\n    return fn(*args, **kwargs)\n  File \"/fate/python/fate_flow/operation/job_saver.py\", line 107, in create_job_family_entity\n    raise Exception(\"Create {} failed:\\n{}\".format(entity_model, e))\nException: Create <Model: Task> failed:\nConstraintError: UNIQUE constraint failed: t_task.f_job_id, t_task.f_task_id, t_task.f_task_version, t_task.f_role, t_task.f_party_id\n"}

[INFO] [2021-06-13 17:46:58,526] [7:139673186752256] - api_utils.py[line:131]: remote http api response: /v1/initiator/202106131743010453261/guest/9999/rerun {'retcode': 100, 'retmsg': 'Traceback (most recent call last):\n  File "/usr/local/lib/python3.6/site-packages/playhouse/apsw_ext.py", line 112, in execute_sql\n    cursor.execute(sql, params or ())\n  File "src/cursor.c", line 236, in resetcursor\napsw.ConstraintError: ConstraintError: UNIQUE constraint failed: t_task.f_job_id, t_task.f_task_id, t_task.f_task_version, t_task.f_role, t_task.f_party_id\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/fate/python/fate_flow/operation/job_saver.py", line 99, in create_job_family_entity\n    rows = obj.save(force_insert=True)\n  File "/fate/python/fate_arch/storage/metastore/base_model.py", line 89, in save\n    return super(BaseModel, self).save(*args, **kwargs)\n  File "/usr/local/lib/python3.6/site-packages/peewee.py", line 5964, in save\n    self.insert(**field_dict).execute()\n  File "/usr/local/lib/python3.6/site-packages/peewee.py", line 1737, in inner\n    return method(self, database, *args, **kwargs)\n  File "/usr/local/lib/python3.6/site-packages/peewee.py", line 1808, in execute\n    return self._execute(database)\n  File "/usr/local/lib/python3.6/site-packages/peewee.py", line 2512, in _execute\n    return super(Insert, self)._execute(database)\n  File "/usr/local/lib/python3.6/site-packages/peewee.py", line 2275, in _execute\n    cursor = database.execute(self)\n  File "/usr/local/lib/python3.6/site-packages/peewee.py", line 2888, in execute\n    return self.execute_sql(sql, params, commit=commit)\n  File "/usr/local/lib/python3.6/site-packages/playhouse/apsw_ext.py", line 112, in execute_sql\n    cursor.execute(sql, params or ())\n  File "/usr/local/lib/python3.6/site-packages/peewee.py", line 2666, in __exit__\n    reraise(new_type, new_type(*exc_args), traceback)\n  File "/usr/local/lib/python3.6/site-packages/peewee.py", line 179, in reraise\n    raise value.with_traceback(tb)\n  File "/usr/local/lib/python3.6/site-packages/playhouse/apsw_ext.py", line 112, in execute_sql\n    cursor.execute(sql, params or ())\n  File "src/cursor.c", line 236, in resetcursor\npeewee.IntegrityError: ConstraintError: UNIQUE constraint failed: t_task.f_job_id, t_task.f_task_id, t_task.f_task_version, t_task.f_role, t_task.f_party_id\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app\n    response = self.full_dispatch_request()\n  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request\n    rv = self.handle_user_exception(e)\n  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception\n    reraise(exc_type, exc_value, tb)\n  File "/usr/local/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise\n    raise value\n  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request\n    rv = self.dispatch_request()\n  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request\n    return self.view_functions[rule.endpoint](**req.view_args)\n  File "/fate/python/fate_flow/scheduling_apps/initiator_app.py", line 48, in rerun_job\n    component_name=request.json.get("component_name"))\n  File "/fate/python/fate_flow/scheduler/dag_scheduler.py", line 363, in rerun_job\n    JobController.initialize_tasks(job_id, _role, _party_id, False, job.f_initiator_role, job.f_initiator_party_id, RunParameters(**job.f_runtime_conf_on_party["job_parameters"]), dsl_parser, component_name=task.f_component_name, task_version=task.f_task_version)\n  File "/fate/python/fate_flow/controller/job_controller.py", line 192, in initialize_tasks\n    role=role, party_id=party_id, run_on_this_party=run_on_this_party, task_info=task_info)\n  File "/fate/python/fate_flow/controller/task_controller.py", line 52, in create_task\n    JobSaver.create_task(task_info=task_info)\n  File "/fate/python/fate_flow/operation/job_saver.py", line 36, in create_task\n    return cls.create_job_family_entity(Task, task_info)\n  File "/usr/local/lib/python3.6/site-packages/peewee.py", line 394, in inner\n    return fn(*args, **kwargs)\n  File "/fate/python/fate_flow/operation/job_saver.py", line 107, in create_job_family_entity\n    raise Exception("Create {} failed:\\n{}".format(entity_model, e))\nException: Create <Model: Task> failed:\nConstraintError: UNIQUE constraint failed: t_task.f_job_id, t_task.f_task_id, t_task.f_task_version, t_task.f_role, t_task.f_party_id\n'}
